app/src/main/java/com/parasoft/demoapp/component/OrderDialog.java

```diff
@@ -78,6 +80,7 @@ public class OrderDialog extends DialogFragment {
     private String responseValue;
     private View contentDivider;
     private EditText commentsField;
+    private OrderStatusRequest orderStatusRequest;
```

Comment:
Instead of create a shared OrderStatusRequest object here, only create when preparing for a call.
So please create local variable in Line 179 and Line 399.